tests/isolated/highlights.vader

```diff
@@ -158,7 +158,7 @@ Execute (cargo error message):
   call neomake#Make(1, [cargo])
   NeomakeTestsWaitForFinishedJobs
 
-  let bufnr = bufnr('%')
+  let bufnr = bufnr('lib.rs')
```

Comment:
Does this pick up the same bufnr?  (it uses `file build/lib.rs` above)

btw: `Log neomake#utils#redir('ls!')` can be handy in this context.

Reply:
`build/lib.rs` does also work and looks more correct.